src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -100,235 +60,430 @@
+        access-type (get-access-type (:access permissions))]
+    (when-not (or joined? pending? (#{:unknown-access :invite-only} access-type))
+      [token-requirements community-id role-permissions? color tags?])))
 
 (defn- status-tag
-  [community-id joined]
+  [{:keys [community-id joined? info-styles]}]
   (let [pending? (rf/sub [:communities/my-pending-request-to-join community-id])]
-    (when (or pending? joined)
-      [rn/view {:style {:position :absolute :top 12 :right 12}}
+    (when (or pending? joined?)
+      [reanimated/view {:style [style/status-tag-position info-styles]}
        [quo/status-tag
-        {:status {:type (if joined :positive :pending)}
-         :label  (if joined
-                   (i18n/label :t/joined)
-                   (i18n/label :t/pending))}]])))
+        {:status {:type (if joined? :positive :pending)}
+         :label  (if joined? (i18n/label :t/joined) (i18n/label :t/pending))}]])))
 
-(defn- add-handlers
+(defn- category-divider
   [community-id
-   joined-or-spectated
-   {:keys [id locked?]
-    :or   {locked? false}
-    :as   chat}]
-  (cond-> chat
-    (and (not locked?) id)
-    (assoc :on-press      (when joined-or-spectated
-                            (fn []
-                              (rf/dispatch [:dismiss-keyboard])
-                              (debounce/throttle-and-dispatch
-                               [:communities/navigate-to-community-chat (str community-id id)]
-                               1000)))
-           :on-long-press #(rf/dispatch
-                            [:show-bottom-sheet
-                             {:content (fn []
-                                         [chat-actions/actions chat false])}])
-           :community-id  community-id)))
-
-(defn- add-handlers-to-chats
-  [community-id joined-or-spectated chats]
-  (mapv (partial add-handlers community-id joined-or-spectated) chats))
-
-(defn- add-handlers-to-categorized-chats
-  [community-id categorized-chats joined-or-spectated]
-  (let [add-on-press (partial add-handlers-to-chats community-id joined-or-spectated)]
-    (map (fn [[category v]]
-           [category (update v :chats add-on-press)])
-         categorized-chats)))
-
-(defn- community-header
-  [title logo description]
-  [quo/text-combinations
-   {:container-style                 {:margin-top
-                                      (if logo
-                                        12
-                                        (+ scroll-page.style/picture-radius
-                                           scroll-page.style/picture-border-width
-                                           12))
-                                      :margin-bottom 12}
-    :avatar                          logo
-    :title                           title
-    :title-number-of-lines           2
-    :description                     description
-    :title-accessibility-label       :community-title
```

Comment:
Hi @ulisesmac, could you please return this label, because it's used in e2e tests. Thanks!